client/state/jetpack-settings/settings/test/reducer.js

```diff
@@ -84,9 +84,9 @@ describe( 'reducer', () => {
 					12345678: SETTINGS_FIXTURE[ siteId ]
 				},
 				action = {
-					type: JETPACK_SETTINGS_RECEIVE,
```

Comment:
@tyxla is this change here doing the right thing? Looks to me that this test was thought for this case, but referencing the wrong action type, but I'm not 100% sure